packages/main/src/themes/TabContainer.css

```diff
@@ -30,6 +30,14 @@
 	border-top: var(--_ui5_tc_header_border_bottom);
 }
 
+:host([header-background-design="Solid"]) .ui5-tc__header {
+	background-color: var(--sapObjectHeader_Background);
+}
+
+:host([header-background-design="Transparent"]) .ui5-tc__header {
+	background-color: transparent;
```

Comment:
In Horizon, the spec implies that there is a box shadow on the Header, which should be turned off for a Transparent header.

https://wiki.one.int.sap/wiki/pages/viewpage.action?pageId=2698913732#IconTabBar(Horizon)-TabBarBackgroundbackground